index.md

```diff
@@ -7,60 +7,46 @@ permalink:
 
 # Hello Decentralized World
 {: .fs-8 }
-<img style="float: right; padding:10px;" src="assets/images/helloworldsquare.png">
+<img style="float:right; padding:6px; width:529px;" src="assets/images/wasm_sample_code.png">
 
-We want to show you how easy it is to build on Tupelo.
-{: .fs-5 .fw-300 }
+Tupelo is easy.
 
-Start building now by adding a [“Hello World” message](tutorials) and get it signed in seconds.
-{: .fs-5 .fw-300 }
-Need more information first?  
-Read below to see how we’re different or get access to important documentation.
-{: .fs-5 .fw-300 }
+The straightforward API and growing list of familiar language libraries make integrating
+the power of a distributed ledger into your application fast and simple.
+{: .fs-3 .fw-300 }
 
-Want to jump right in and build?  Start accessing
-Tupelo using the new **[Tupelo WASM SDK](https://www.tupelo.org/blog/2019/8/22/new-wasm-based-sdk-for-tupelo) right from the browser.**
-{: .fs-5 .fw-300 }
+You can read all about Tupelo below, or if you prefer to learn by doing, **jump in and try our
+new [Tupelo WASM SDK](https://www.tupelo.org/blog/2019/8/22/new-wasm-based-sdk-for-tupelo)**.
+The WASM SDK enables you to easily leverage the power of trust that Tupelo
+makes possible right from a browser.
+{: .fs-3 .fw-300 }
 
+Another option is to run through our [“Hello World” tutorials](tutorials) and have your
+first transaction signed in minutes. That is a total of minutes to set up and submit the
+request, the Tupelo TestNet will sign your transaction in less than a second.
+{: .fs-3 .fw-300 }
+***
 # We Are Distributed Ledger Done Differently
```

Comment:
Tupelo is Distributed Ledger Done Differently

Reply:
Doh, missed in title.  Fixorated.